include/deal.II/non_matching/mesh_classifier.h

```diff
@@ -56,7 +57,8 @@ namespace NonMatching
     inside,
     outside,
     intersected,
-    unassigned
+    unassigned,
+    aligned
```

Comment:
Shall we add a comment/assert that a cell should never be aligned?

Reply:
I think there should be an assert. I will add it.